airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_check.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env bash
 set -o errexit -o nounset -o pipefail
 
-# Check if airbyte-ci is on the PATH and pointing to the correct location
+echo "Checking if airbyte-ci is correctly installed..."
+
+# Check that airbyte-ci is on the PATH
+# If not print an error message and exit
+if ! which airbyte-ci >/dev/null 2>&1; then
+    echo "airbyte-ci is not on the PATH"
+    echo "Check that \$HOME/.local/bin is part of the PATH"
+    echo ""
+    echo "If not, please add 'export PATH=\"\$HOME/.local/bin:\$PATH\"' to your shell profile"
+    exit 1
+fi
+
 EXPECTED_PATH="$HOME/.local/bin/airbyte-ci"
 AIRBYTE_CI_PATH=$(which airbyte-ci 2>/dev/null)
-
 if [ "$AIRBYTE_CI_PATH" != "$EXPECTED_PATH" ]; then
-    echo "airbyte-ci is either not on the PATH or not pointing to $EXPECTED_PATH"
-    echo "Check that airbyte-ci exists at $HOME/.local/bin and $HOME/.local/bin is part of the PATH"
-    echo "If it does, try running 'make tools.airbyte-ci.clean', then run 'make tools.airbyte-ci.install' again"
+    echo "airbyte-ci is not from the expected install location: $EXPECTED_PATH"
+    echo "airbyte-ci is installed at: $AIRBYTE_CI_PATH"
+    echo "Check that airbyte-ci exists at \$HOME/.local/bin and \$HOME/.local/bin is part of the PATH"
+    echo ""
+    echo "If it is, try running 'make tools.airbyte-ci.clean', then run 'make tools.airbyte-ci.install' again"
```

Comment:
Would baking `clean` into the install (and maybe even the update) to get rid of all the old binaries running around be an option we'd consider?

Reply:
This, and the thread around SSL updating that I've gotten 2 asks about since we fixed it, both read as "we installed/updated it successfully, but we're not referencing that version" - feels like we could handle this ourselves under the hood

Reply:
Thinking about this more. I think this is a good idea.

But theres a catch. Until we either 
1. convert clean to a python file
2. or include it in the pyinstaller bundle come how.

auto upgrade wont automatically clean.

But i think thats a minor nice to have. And clean as part of the makefile install is a good idea.

Reply:
Sounds good - let's do the easy stuff then (clean as part of makefile) and punt on trying to do it in auto-upgrade!

Reply:
Also with regards to the "don't mess with my machine" that we talked about - I think cleanup on install is a pretty common theme, but cleanup on autoupgrade less so